_posts/2023-01-05-introducing-arrow-adbc.md

```diff
@@ -144,7 +144,7 @@ ADBC fills a specific niche that related projects do not address. It is both:
     <tr>
       <th></th>
       <th class="align-top" style="width: 40%" scope="col">Vendor-neutral (database APIs)</th>
-      <th class="align-top" style="width: 40%" scope="col">Vendor-specific (database protocols)</th>
+      <th class="align-top" style="width: 40%" scope="col">Database protocols</th>
```

Comment:
I think it's still fair to call them vendor-specific; after all, multiple databases also use the PostgreSQL protocol (it just doesn't have a generic name). Maybe "varies by vendor (database protocols)"?

Reply:
Maybe a better phrase would be "Database specific protocols"